HIRS_Utils/src/main/java/hirs/data/persist/certificate/IssuedAttestationCertificate.java

```diff
@@ -40,6 +41,18 @@
         public Selector(final CertificateManager certificateManager) {
             super(certificateManager, IssuedAttestationCertificate.class);
         }
+
+        /**
+         * Specify a device id that certificates must have to be considered
+         * as matching.
+         *
+         * @param device the device id to query
+         * @return this instance (for chaining further calls)
+         */
+        public Selector byDeviceId(final UUID device) {
```

Comment:
What would be the time to write up a test for this in `DBCertificateManagerTest`? That seems to be where the Selector for this class is tested.

Reply:
I added one test for now. If we wanted to we could add more